modules/ROOT/pages/multi-factor-authentication.adoc

```diff
@@ -0,0 +1,143 @@
+
+Built-in authenticator:: Registers a physical authentication device, such as Touch ID, to verify your identity when logging in to Anypoint Platform.
+
+Security key:: Registers a USB security key to your account. The device is then authorized to create verification codes that you provide when logging in to Anypoint Platform.
+
+Salesforce Authenticator:: Registers the Salesforce Authenticator mobile app to create verification codes that you provide when logging in to Anypoint Platform.
+
+== Enable Multi-factor Authentication for All User Accounts
+
+For enhanced security, you can require all users in your organization to use multi-factor authentication. Users must configure their authentication method when they log in to Anypoint Platform after you initially enable multi-factor authentication.
+
+[NOTE]
+--
+To prevent service disruptions, identify any accounts that are used to programmatically call Anypoint Platform, or _service accounts_. You must either configure these calls using the xref:connected-apps-overview.adoc[Connected Apps] feature or add these accounts to the *Exempt Accounts* section when configuring multi-factor authentication requirements. 
+--
+
+. Log in to Anypoint Platform using an account that has the Organization Administrator permission.
+. Navigate to *Access Management*.
+. In the navigation menu, click *Multi-factor auth*.
+. Select *Required*.
+. If you want to exclude users, such as service accounts, from multi-factor authentication, add users to the *Exempt Accounts* section.
+. Click *Save*. +
+All users in your organization are now required to configure a multi-factor authentication method when they log into Anypoint Platform. +
+If a user already has multi-factor authentication configured for their account, they are not prompted to configure another method.
+
+=== Exempt Users from Multi-factor Authentication
+
+Accounts that make programmatic calls to Anypoint Platform, or service users, should be exempt from multi-factor authentication since they cannot provide verification aside from a username and password.
+
+As a best practice, MuleSoft recommends using the xref:connected-apps-overview.adoc[Connected Apps] feature to authorize apps to make programmatic calls to Anypoint Platform instead of using individual user accounts. Using the Connected Apps feature provides more security and control than using user accounts.
+
+. Log in to Anypoint Platform using an account that has the Organization Administrator permission.
+. Navigate to *Access Management*.
+. In the navigation menu, click *Multi-factor Auth*.
+. In the *Exempt Accounts* section, add the accounts that you want to exempt. +
+Only users who do not use single sign-on (SSO) appear in this list.
+. Click *Save*. +
+The exempt user accounts appear in the *Exempt Accounts* section.
+
+You can remove user accounts from *Exempt Accounts* list by clicking *X* next to their account name.
+
+== Enable Multi-factor Authentication on Individual User Accounts
```

Comment:
"== Enable Multi-factor Authentication on Your User Account" maybe be more accurate?

Reply:
```suggestion
== Enable Multi-factor Authentication on Your User Account
```